tests/components/xiaomi_ble/test_config_flow.py

```diff
@@ -0,0 +1,98 @@
+"""Test the Xiaomi config flow."""
+
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.xiaomi_ble.const import DOMAIN
+from homeassistant.data_entry_flow import FlowResultType
+
+from . import MMC_T201_1_SERVICE_INFO, LYWSDCGQ_SERVICE_INFO, NOT_SENSOR_PUSH_SERVICE_INFO
```

Comment:
```suggestion
from . import LYWSDCGQ_SERVICE_INFO, MMC_T201_1_SERVICE_INFO, NOT_SENSOR_PUSH_SERVICE_INFO
```

alphabetic order to fix the PR tests